docs/reference/indices/recovery.asciidoc

```diff
@@ -20,6 +20,10 @@ To see cluster-wide recovery status simply leave out the index names.
 GET /_recovery?human
 --------------------------------------------------
 // CONSOLE
+// TEST[s/^/POST \/_snapshot\/my_repository\/snap_1\/_restore\?wait_for_completion=true\n/]
+// TEST[s/^/DELETE index1\n/]
+// TEST[s/^/PUT \/_snapshot\/my_repository\/snap_1\?wait_for_completion=true\n/]
+// TEST[s/^/PUT \/_snapshot\/my_repository\n{"type": "fs","settings": {"location": "recovery_asciidoc" }}\n/]
```

Comment:
maybe you could have a commented snippet (in the asciidoc sense) that does the setup and here just do a TEST[continued]? We do that sometimes to test responses without having to actually put the response in the docs. I'm wondering whether the same trick can be used to quick setups too.